pkgs/development/octave-modules/video/default.nix

```diff
@@ -18,8 +20,20 @@ buildOctavePackage rec {
     hash = "sha256-fn9LNfuS9dSStBfzBjRRkvP50JJ5K+Em02J9+cHqt6w=";
   };
 
+  preBuild = ''
+    pushd src
```

Comment:
This likely does not work because sourceRoot is not set correctly. Also bootstrap and configure definitely belong into the configurePhase. Maybe we should just use autoreconfHook?